src/components/big-interactive-pages/editor.tsx

```diff
@@ -33,7 +34,9 @@ interface ResizeState {
 
 const minOutputAreaSize = 360
 const defaultOutputAreaSize = 400
-const heightMargin = 130
+const widthMargin = 130
+
+const minHelpAreaSize = 200
```

Comment:
nit - Suggest adding comments to all of these constants describing exactly what they represent (minHelpAreaSize - is it width or height, for instance)